examples/grpc/proxying/client/fetch-proto.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
DRY: We can refer the file at scripts/ dir instead of copying it here.

Reply:
The fetch-proto was not designed for this as this is example specific. We could make it abstract, but it might go beyond the scope of this PR?